components/azure-storage/get-storage-account-id.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
since this script is specific to AZURE_STORAGE_ACCOUNT_GEN2, put -hfs in its name?